Jenkinsfile

```diff
@@ -6,8 +6,7 @@ def master_branches = ["master", ] as String[]
 
 pipeline {
   agent {
-    dockerfile {
-      args  "--shm-size=1g"
+    label 'dcos-ui'
```

Comment:
we need to build dockerfile here for dependencies, if you want to change the label - thats defined in jenkins config.

also we want to use dockerfile do be able to eaily change it, docker labels are kinda hardcoded and we had this before… it was a pain…